docs/tutorial/index.md

```diff
@@ -354,53 +202,46 @@ ssl_public_key = C:\Users\<YOUR_WINDOWS_USER_NAME>\landscape_server.pem
 ```
 % If really needed we can start without the SSL public key and add it after the Windows host is registered in Landscape.
 
-
-Then load that file using the "Custom Configuration" part of the the screen, as shown below:
+Then load that file using the "Custom Configuration" part of the screen, as shown below:
 
 ![Loading Landscape custom config](./assets/loading-custom-landscape-config.png)
 
-Click on the "Continue" button. You'll see a status screen confirming the configuration is complete.
+Click on the "Continue" button and you will see a status screen confirming the configuration is complete.
 
 ![Configuration is complete](./assets/status-complete.png)
 
-Done! You can close the UP4W window if you want.
+Done! You can now close the UP4W window.
 
-This has attached your Ubuntu Pro subscription to UP4W on the Windows host; UP4W will automatically forward it to the Ubuntu Pro client on your Ubuntu WSL instances; thus, all of your Ubuntu WSL instances will be automatically added to your Ubuntu Pro subscription.
+Your Ubuntu Pro subscription is now attached to UP4W on the Windows host.
+UP4W will automatically forward the subscription to the Ubuntu Pro client on your Ubuntu WSL instances.
+This means that all Ubuntu WSL instances will be automatically added to your Ubuntu Pro subscription.
 
 This has also configured the Landscape client built into your UP4W Windows agent to know about your Landscape server; UP4W will forward this configuration to the Landscape client on your Ubuntu WSL instances as well; and all systems where the Landscape client has been configured this way are automatically registered with Landscape.
 
+### UP4W host registration
```

Comment:
```suggestion
### UP4W host registration with Landscape
```

Stopping at the word registration feels a bit obscure to me: registration with/into what/where? The screenshots might be enough for the users acquainted with Landscape, but I don't think we should rely on that knowledge.

Reply:
Good point. In general, I was trying to shorten excessively long headers while highlighting the purpose of a given section. This tutorial also has a lot of references to different technology (UP4W, WSL, Ubuntu, Landscape,..), which could distract from what is being achieved in practice with UP4W.

In this case I agree that "with Landscape" should be preserved.